src/components/forms/Scopes/Scopes.vue

```diff
@@ -1,25 +1,30 @@
 <template>
```

Comment:
new design have some new points, but in this pr they are not implemented:
- logo in header
- tooltips for doc type
- header of page just `Permissions`, not `Requested Permissions`

![image](https://user-images.githubusercontent.com/756731/82530763-b2eb4300-9b46-11ea-870d-61f58f4ae959.png)

Reply:
Tooltips -- ok
Title is correct, see task description

Reply:
title is correct, yes, my fault